src/rail/estimation/algos/deepdisc.py

```diff
@@ -161,17 +170,18 @@ class DeepDiscInformer(CatInformer):
         output_dir=Param(str, "./", required=False, msg="The directory to write output to."),
         output_name=Param(str, "deepdisc_informer", required=False, msg="What to call the generated output."),
         chunk_size=Param(int, 100, required=False, msg="Chunk size used within detectron2 code."),
+        training_percent=Param(float, 0.8, required=False, msg="The percentage of the input data to use for training. The remaining percent is used for evaluation."),
```

Comment:
The description here is a little wordy, any suggestions to make it more clear would be appreciated.

Reply:
The fraction of input data used to split into training/evaluation sets